perfmetrics/scripts/continuous_test/gcp_ubuntu/build.sh

```diff
@@ -36,16 +35,26 @@ MOUNT_POINT=gcs
 # The VM will itself exit if the gcsfuse mount fails.
 gcsfuse $GCSFUSE_FLAGS $BUCKET_NAME $MOUNT_POINT
 
-# Executing perf tests
-chmod +x run_load_test_and_fetch_metrics.sh
-./run_load_test_and_fetch_metrics.sh
+chmod +x perfmetrics/scripts/run_load_test_and_fetch_metrics.sh
+./perfmetrics/scripts/run_load_test_and_fetch_metrics.sh
 # Copying gcsfuse logs to bucket
 gsutil -m cp $LOG_FILE gs://gcs-fuse-dashboard-fio/fio-gcsfuse-logs/
 
 # Deleting logs older than 10 days
-python3 utils/metrics_util.py gcs/fio-gcsfuse-logs/ 10
+python3 perfmetrics/scripts/utils/metrics_util.py gcs/fio-gcsfuse-logs/ 10
 
 # ls_metrics test
-cd "./ls_metrics"
-chmod +x run_ls_benchmark.sh
-./run_ls_benchmark.sh
+chmod +x perfmetrics/scripts/ls_metrics/run_ls_benchmark.sh
+./perfmetrics/scripts/ls_metrics/run_ls_benchmark.sh
+sudo umount gcs
+
+# Executing implicitdir integration tests for the testbucket flag
+GODEBUG=asyncpreemptoff=1 go test ./tools/integration_tests/implicitdir/ -v --testbucket=gcsfuse-implicitdir-tests
```

Comment:
same comments as other file.

Reply:
because for readonly tests we will have different bucket to tests.